cmd/jaeger/config-query.yaml

```
@@ -0,0 +1,41 @@
+service:
+  extensions: [jaeger_storage, jaeger_query, healthcheckv2]
+  pipelines:
+    traces:
+      receivers: [nop]
+      processors: [batch]
+      exporters: [nop]
+  telemetry:
+    resource:
+      service.name: jaeger-query
+    metrics:
+      level: detailed
+      address: 0.0.0.0:8888
```

```suggestion
      address: 0.0.0.0:8889
```
otherwise it will conflict with collector if run simultaneously. Also need to change the port for healthcheck

---

> otherwise it will conflict with collector if run simultaneously. Also need to change the port for healthcheck

i think healthcheck uses  default port 16687 , that  specifically designated for admin operations including healthchecks  , can you please eleborate .